R/run.R

```diff
@@ -35,12 +36,20 @@
 #' @importFrom withr with_output_sink
 #' @export
 run <- function(renvInstallPackages = NULL,
-                computations = baseComputations[c("magpiePrep", "remindPrep")],
+                computations = c("magpiePrep", "remindPrep"),
                 piktestsFolder = getwd(),
                 runFolder = NULL,
                 jobNameSuffix = "",
                 executionMode = c("slurm", "directly"),
                 localCache = TRUE) {
+
+  if (!is.list(computations)) {
+    if (all(computations %in% names(baseComputations))) {
+      computations <- baseComputations[computations]
+    } else {
+      stop("Computations provided in an incompatible format!")
```

Comment:
This reads to me like "something went wrong" :stuck_out_tongue: 
```suggestion
      stop("Unknown computations provided: [", paste(setdiff(computations, names(baseComputations)), collapse = ", "),
           "] - Available computations: [", paste(names(baseComputations), collapse = ", "), "]")
```